### PR TITLE
UCT/IB/UD: Improve AV logging and show dgid/rmac

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7095/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@brminich pls take a look

File: src/uct/ib/mlx5/ib_mlx5_log.c [hunk 1]
```diff
@@ -174,17 +174,45 @@ static uint64_t network_to_host(void *ptr, int size)
...
+    p += strlen(p);
+
+    if (!is_eth) {
+        snprintf(p, endp - p, " rlid %d", ntohs(base_av->rlid));
+        p += strlen(p);
+    }
 
     if (mlx5_av_base(&dgseg->av)->dqp_dct & UCT_IB_MLX5_EXTENDED_UD_AV) {
+        grh_av = mlx5_av_grh(&dgseg->av);
+        if (is_eth || (ntohl(grh_av->grh_gid_fl) & UCS_BIT(30))) {
```
Comment (brminich) [inline src/uct/ib/mlx5/ib_mlx5_log.c]: I'd add comment what 30-th bit means , it takes time to check it in the driver)

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7118/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7123/ for details (Mellanox internal link).

**Pull request merged**